src/status_im/contexts/wallet/wallet_connect/utils/data_store.cljs

```diff
@@ -73,3 +73,25 @@
   (-> db
       get-db-current-request-event
       get-request-params))
+
+(defn get-network-from-request
+  [db]
+  (-> db
+      get-db-current-request-event
+      (get-in [:params :chainId])))
+
+(defn get-networks-from-proposal
+  "Find networks from current proposal as a string of comma separated values"
+  [db & [proposal]]
+  (let [current-proposal (get-in db [:wallet-connect/current-proposal :request])]
+    (cond
+      current-proposal
+      (->> (get-in db [:wallet-connect/current-proposal :session-networks])
+           vec
+           (string/join ","))
```

Comment:
does mixpanel only allow string values, or can we also send a json array?

Reply:
yes, 
<img width="360" alt="image" src="https://github.com/user-attachments/assets/e63a8d66-7c3c-435e-83f4-c46a6779eb32">

Reply:
then would be better to use arrays, so we can query the networks from the dashboard